### PR TITLE
add the ability to serve a font directory

### PR DESCRIPTION
**Pull request by ldgeo**
Add an endpoint for serving glyphs following the mapbox gl specification: 

https://docs.mapbox.com/mapbox-gl-js/style-spec/#root-glyphs. 
https://github.com/openmaptiles/fonts

Mostly useful in development.

### CODE REVIEW

File: postile/postile.py [hunk 1]
```diff
@@ -117,6 +119,14 @@ def prepared_query(filename):
         headers={"Content-Type": "application/json"}
     )
 
+@app.route('/fonts/<fontstack:string>/<frange:string>.pbf')
+async def get_fonts(request, fontstack, frange):
+    if not Config.fonts:
+        return response.text('no fonts available', status=404)
+    return await response.file(
+        Path(Config.fonts) / fontstack / f'{frange}.pbf',
```
Comment (elemoine) [inline postile/postile.py:127]: Do we use f-strings already? Do we care about Python < 3.6?

Comment (ldgeo) [reply postile/postile.py:127]: Yes, f-strings are already used in code, and Python3.6 is mentioned as a requirement in the readme.

**Comment by elemoine** [comment]
LGTM

**Comment by ldgeo** [comment]
Thanks @elemoine

**Pull request merged**